### PR TITLE
feat: Added privacy statement

### PR DESCRIPTION
**Pull request by daviroo**
## Pull Request checklist

Please review the below PR requirements:

- Build was run locally and any changes were pushed
- Ensure that the title clearly describes the changes
- Issue to be referenced in the PR description rather than in the commits or PR title
- Clean commit history

### CODE REVIEW

File: app/[lang]/about/page.tsx [hunk 1]
```diff
@@ -86,6 +86,32 @@ export default function About({ params }) {
...
+          </h3>
+          <p className="mt-2 font-nunito text-xl">
+            {t('about.privacy.paragraph_two')}
+          </p>
+          <h3 className="mt-4 text-2xl sm:text-3xl">
+            {t('about.privacy.sub_heading_two')}
+          </h3>
+          <p className="mt-2 font-nunito text-xl">
+            {t('about.privacy.paragraph_three')}
+          </p>
```
Comment (benalleng) [inline app/[lang]/about/page.tsx:105]: This `<p>` tag cannot be the parent of the `<ol>`

File: i18n/locales/en.ts [hunk 2]
```diff
@@ -65,6 +65,25 @@ const translations = {
...
+      title: 'Our commitment to privacy',
+      paragraph_one:
+        'At Saving Satoshi, we are deeply committed to ensuring the privacy and security of our users. We believe in transparency and want you to understand how and why we track app usage. To achieve this, we use a powerful tool called <Link href="https://umami.is/" className="underline" target="_blank">Umami</Link>.',
+      sub_heading_one: 'What is Umami?',
+      paragraph_two:
+        'Umami is an open-source analytics platform that helps us gather essential insights about how you use our app. It allows us to make data-driven decisions and continuously enhance our product to better meet your needs.',
+      sub_heading_two: 'What Data Do We Collect?',
+      paragraph_three: `Rest assured, we only collect non-personal and anonymized data, such as: 
+      <ol>
+      <li><span className="font-bold">Product Improvement:</span> We use the data to identify areas where our app can be improved. By understanding how you use our app, we can make it more efficient, user-friendly, and secure.</li>
```
Comment (benalleng) [inline i18n/locales/en.ts:78]: upon testing, it seems to be favoring rendering this span element before the list, upon removing the span I was able to get it to work marginally better

File: i18n/locales/en.ts [hunk 3]
```diff
@@ -65,6 +65,25 @@ const translations = {
...
+      <li><span className="font-bold">Compatibility:</span>Understanding the devices and platforms you use allows us to optimize our app for various configurations, ensuring it works seamlessly for you.</li>
+      </ol>`,
```
Comment (benalleng) [inline i18n/locales/en.ts:80]: It also really doesn't seem to play nivcely with having these line breaks, it works better if there are no line-breaks

**Review by benalleng — CHANGES_REQUESTED** [review]
I think the issue seems to be three-fold: the addition of the `<span>` element inside of the `<ol>` tag seems to be causing the most problems but as well the `<ol>` is inside of a `<p>` tag causes errors. Lastly the line-breaks seem to make the regex not able to recognize it

File: lib/translate.tsx [hunk 4]
```diff
@@ -153,6 +156,30 @@ function injectComponent(result, type) {
...
+            ? liMatches.map((liHtml, itemIndex) => {
+                const liLabel = getFirstMatch(liHtml, labelRegex)
+                const liClassName = getFirstMatch(liHtml, classNameRegex)
+                return (
+                  <li key={`item-${itemIndex}`} className={liClassName}>
+                    {liLabel}
+                  </li>
+                )
+              })
+            : []
```
Comment (benalleng) [inline lib/translate.tsx:172]: I think the `<span>` element is recognized and rendered before everything inside the `<li>` tag is rendered so thats why removing the `<span>` element allows it to work.

**Comment by daviroo** [comment]
@benalleng I changed my approach entirely so I've marked the other comments as resolved.

**Comment by daviroo** [comment]
@adamjonas do you have any comment on the copy?
image

**Review by benalleng — APPROVED** [review]

**Pull request merged**